src/main/java/picard/sam/SamErrorMetric/BaseErrorCalculator.java

```diff
@@ -36,8 +38,16 @@
      **/
     @Override
     public void addBase(final SamLocusIterator.RecordAndOffset recordAndOffset, final SamLocusAndReferenceIterator.SAMLocusAndReference locusAndRef) {
-        if (!SequenceUtil.isNoCall(recordAndOffset.getReadBase())) {
-            nBases++;
+        // TODO mgatzen should deletions be counted towards total bases?
```

Comment:
I personally think that only read-bases should count towards the total bases, which will be used as the denominator

Reply:
I agree - thank you